app/uk/gov/hmrc/eoricommoncomponent/frontend/views/has_existing_eori.scala.html

```diff
@@ -36,7 +37,7 @@ <h1 class="govuk-heading-l">@messages("cds.has-existing-eori.heading")</h1>
     <p id="para2" class="govuk-body">@messages("cds.has-existing-eori.para2")</p>
     <p id="para3" class="govuk-body">@messages("cds.has-existing-eori.para3")</p>
 
-    @helpers.form(uk.gov.hmrc.eoricommoncomponent.frontend.controllers.routes.HasExistingEoriController.enrol(service)) {
+    @helpers.form(uk.gov.hmrc.eoricommoncomponent.frontend.controllers.routes.EmailController.form(service, subscribeJourney = SubscribeJourney(AutoEnrolment))) {
```

Comment:
Good question. Actually, this is has to be changed. `has_existing_eori` page is the beginning for the auto-enrolment journey.

I've changed this so that on CDS subscription users get redirected next to EmailController, where we start Email capture journey; and for non CDS subscriptions this stays the same without Email capture.